android/login/src/main/kotlin/kmp/android/login/ui/AuthScreen.kt

```diff
@@ -131,37 +130,37 @@ private fun AuthForm(
 ) {
     val titleText = stringResource(
         if (screen == AuthScreen.Login) {
-            cz.matee.devstack.kmp.android.shared.R.string.login_view_headline_title
+            kmp.android.shared.R.string.login_view_headline_title
```

Comment:
Same as above, you could import the R (it was not imported before I think because of refactoring).